run/test/specs/onboarding_pp.spec.ts

```diff
@@ -30,13 +28,11 @@ async function onboardingPP(platform: SupportedPlatformsType) {
     const retrievedURL = await device.getTextFromElement(urlField)
     // Add https:// to the retrieved URL if the UI doesn't show it (Chrome doesn't, Safari does)
     const fullRetrievedURL = retrievedURL.startsWith('https://') ? retrievedURL : `https://${retrievedURL}`;
-    console.log(`The URL is "${fullRetrievedURL}"`);
-    console.log(`The expected URL is "${ppURL}"`);
     // Verify that it's the correct URL 
-    if (fullRetrievedURL === ppURL) {
-        console.log("The URLs match.");
-    } else {
+    if (fullRetrievedURL !== ppURL) {
         throw new Error("The retrieved URL does not match the expected.");
```

Comment:
Yeah this is where the actual retrieved URL should be, in the Error log

Reply:
done